catalog-rest-service/src/main/resources/json/schema/entity/teams/team.json

```diff
@@ -68,6 +68,16 @@
     "defaultRoles": {
       "description": "Roles to be assigned to all users that are part of this team.",
       "$ref": "../../type/entityReference.json#/definitions/entityReferenceList"
+    },
+    "owner": {
```

Comment:
@mithmatt this might requires adding to sql files to add the default values in the existing schemas